ignite/services/plugin/plugin_test.go

```diff
@@ -415,6 +417,8 @@ func TestPluginLoad(t *testing.T) {
 }
 
 func TestPluginLoadSharedHost(t *testing.T) {
+	t.Skip() // Re-enable when 0.52 integration finished
```

Comment:
can we create an issue to remember?

Reply:
> can we create an issue to remember?

https://github.com/ignite/cli/issues/4386